gym/envs/mujoco/mujoco_env.py

```diff
@@ -36,7 +36,14 @@ class MujocoEnv(gym.Env):
     """Superclass for all MuJoCo environments.
     """
 
-    def __init__(self, model_path, frame_skip, rgb_rendering_tracking=True):
+    def __init__(self, model_path, frame_skip, rgb_rendering_tracking=None):
+
+        if rgb_rendering_tracking is not None:
+            print("`rgb_rendering_tracking` is deprecated and will be removed"
+                  " in a future version of Gym. To maintain the old behavior,"
+                  " you can omit the `camera_id` and `camera_name` in the"
+                  " `env.render()` method.")
```

Comment:
this message not, technically, quite accurate - I if understand the logic in render method, omitting camera_id and camera_name in render arguments is the same as `rgb_rendering_tracking=True`. However, if the execution got to printing the message above, chances are `rgb_rendering_tracking=False` (because the opposite is default and does not need to be specified explicitly). 
I think we can delete the whole rgb_rendering_tracking logic entirely (and make a new version release out of this).

Reply:
Sounds good to me! Changed in the latest commit.